pkg/transport/manager.go

```diff
@@ -136,6 +143,26 @@ func (tm *Manager) serve(ctx context.Context) {
 	}
 }
 
+// AddTrustedVisors adds transports to trusted visors from config.
+func (tm *Manager) AddTrustedVisors(ctx context.Context) {
```

Comment:
I feel like this does not add enough functionality to be a member here. Instead, I would keep this in the `/pkg/visor/init.go` file and just call `tm.SaveTransport` in a loop.

Reply:
@evanlinjin I was going to put it into the initialization step. However, I decided that trusted visors should be added only after all initialization steps finish with success. If I keep this in `NewVisor`, the function would be too large. 

Let me know if changed your mind. If not, I will move it into the transport initialization step.

Reply:
> If I keep this in NewVisor, the function would be too large.

@nkryuchkov A separate function will be fine. I would, however, put this function in the `visor` package.

> However, I decided that trusted visors should be added only after all initialization steps finish with success.

@nkryuchkov How about we create another `init` function? Maybe `initConnectTrustedVisors` or something, and append it to the end of `initStack()`.